orca/request_opendap.py

```diff
@@ -1,21 +1,30 @@
 import re
-from xarray import open_dataset
+from xarray import open_dataset, load_dataset
+from tempfile import NamedTemporaryFile
 
+from .split import split_url
 
-def request_opendap(url, out_file):
-    try:
-        d = open_dataset(url)
-        d.to_netcdf(out_file)
-    except Exception as e:
-        raise e
 
-    return out_file
+def request_opendap(url):
+    data = open_dataset(url)
+    size = data.nbytes
+
+    if size / 2 > 5e8:
+        urls = split_url(url, size)
+    else:
+        urls = [url]
```

Comment:
You could probably squish this in to a single line. Also it would be helpful to leave a note on what the `5e8` is.